experimental/Schemes/ProjectiveSchemes.jl

```diff
@@ -1014,3 +1014,30 @@ of ``X`` and ``Y``, respectively.
   ff = CoveredSchemeMorphism(X, Y, phi)
   return ff
 end
+
+# Basic functionality required for Warham
+@attr Int64 function dim(P::AbsProjectiveScheme{<:Field})
```

Comment:
```suggestion
@attr Int function dim(P::AbsProjectiveScheme{<:Field})
```
Concrete and does not depend on the architecture.